Dockerfile-alpine315

```diff
@@ -0,0 +1,22 @@
+FROM         alpine:3.15
+MAINTAINER   PcapPlusPlus <pcapplusplus@gmail.com>
+CMD          bash
+
+# Required system packages
+RUN apk add --no-cache \
+  bash \
+  clang \
+  cmake \
+  curl \
+  doxygen \
+  g++ \
+  git \
+  libpcap-dev \
+  make \
+  py3-pip \
+  tar \
+  wget \
+  zstd-dev
+
+# Update python3 pip
+RUN python3 -m pip install --upgrade pip setuptools wheel
```

Comment:
Did you run it in the CI to see if PcapPlusPlus can build and run there?

Reply:
Yes and send a fix for one header